torch_xla/csrc/xla_lower_util.cpp

```diff
@@ -504,6 +504,26 @@ xla::XlaOp BuildDropout(xla::XlaOp input, float probability, xla::XlaOp seed) {
   return input * mask;
 }
 
+std::vector<xla::XlaOp> BuildNativeDropout(xla::XlaOp input, xla::XlaOp seed,
+                                           float probability,
+                                           c10::optional<bool> train) {
+  const xla::Shape& shape = ShapeHelper::ShapeOfXlaOp(input);
+  if (!train.has_value() || *train) {
+    xla::XlaOp prob =
+        XlaHelpers::ScalarBroadcast<float>(probability, shape, input.builder());
+    xla::XlaOp one = xla::One(input.builder(), shape.element_type());
+    xla::XlaOp mask = BuildBernoulli(prob, seed, shape.element_type());
+    mask = mask / (one - prob);
```

Comment:
this line should be in the `if (probability > 0.0f) {` block?

Reply:
Nice catch!